src/main/kotlin/app/revanced/patches/youtube/general/snackbar/ForceSnackbarTheme.kt

```diff
@@ -0,0 +1,109 @@
+package app.revanced.patches.youtube.general.snackbar
+
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.stringPatchOption
+import app.revanced.patches.youtube.utils.compatibility.Constants.COMPATIBLE_PACKAGE
+import app.revanced.patches.youtube.utils.settings.SettingsPatch
+import app.revanced.util.doRecursively
+import app.revanced.util.insertNode
+import app.revanced.util.patch.BaseResourcePatch
+import org.w3c.dom.Element
+
+@Suppress("DEPRECATION", "unused")
+object ForceSnackbarTheme : BaseResourcePatch(
+    name = "Force snackbar theme",
+    description = "Force snackbar background color to match selected theme.",
+    dependencies = setOf(SettingsPatch::class),
+    compatiblePackages = COMPATIBLE_PACKAGE,
+) {
+    private const val BACKGROUND = "?ytChipBackground"
+    private const val STROKE = "none"
+
+    private val CornerRadius by stringPatchOption(
+        key = "CornerRadius",
+        default = "8.0dip",
+        title = "Corner radius",
+        description = "Specify a corner radius for the snackbar."
+    )
+
+    private val BackgroundColor by stringPatchOption(
+        key = "BackgroundColor",
+        default = BACKGROUND,
+        values = mapOf(
+            "Chip" to BACKGROUND,
+            "Base" to "?ytBaseBackground"
+        ),
+        title = "Background color",
+        description = "Specify a background color for the snackbar. You can specify hex color."
```

Comment:
```suggestion
        description = "The background color of the snackbar. Can be a hex color (#AARRGGBB) or a color resource reference."
```

(I don't actually know what a color resource reference, but I'm assuming that the stuff like `?ytBaseBackground` falls into that category. If not, remove that part from my suggestion.)